src/components/results/results.vue

```diff
@@ -2,21 +2,26 @@
   <ResultsList v-if="$x.totalResults" v-infinite-scroll:main-scroll>
     <BannersList>
       <PromotedsList>
-        <component :is="grid" :columns="2" class="x-grid" :animation="staggeredFadeAndSlide">
-          <template #result="{ item: result }">
-            <Result :result="result" data-test="search-grid-result" />
-          </template>
-          <template #banner="{ item: banner }">
-            <MainScrollItem :item="banner" tag="article">
-              <Banner :banner="banner" />
-            </MainScrollItem>
-          </template>
-          <template #promoted="{ item: promoted }">
-            <MainScrollItem :item="promoted" tag="article">
-              <Promoted :promoted="promoted" />
-            </MainScrollItem>
-          </template>
-        </component>
+        <NextQueriesList>
+          <component :is="grid" :columns="2" class="x-grid" :animation="staggeredFadeAndSlide">
+            <template #result="{ item: result }">
+              <Result :result="result" data-test="search-grid-result" />
+            </template>
+            <template #banner="{ item: banner }">
+              <MainScrollItem :item="banner" tag="article">
+                <Banner :banner="banner" />
+              </MainScrollItem>
+            </template>
+            <template #promoted="{ item: promoted }">
+              <MainScrollItem :item="promoted" tag="article">
+                <Promoted :promoted="promoted" />
+              </MainScrollItem>
+            </template>
+            <template #next-queries-group="{ item: { nextQueries } }">
+              <NextQueriesGroup :nextQueries="nextQueries" />
```

Comment:
Not sure if I can wrap this in a `MainScrollItem` because `nextQueries` doesn't have an `id` property. Maybe I should take the `id` of the first next query?

Reply:
That will depend on if we want to "track" the next queries position in the scroll.

Reply:
But in this case we dont need to scroll to the next-queries because there are results next to the next queries so the scroll id will be the one from the result.

Reply:
That's what I thought then 👍  I'll leave it as is.